cmd/minishift/cmd/start_preflight_test.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"testing"
+
+	configCmd "github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/spf13/viper"
+)
+
+func TestIsoUrl(t *testing.T) {
+	var isoURLCheck = []struct {
+		in  string
+		out bool
+	}{
+		{"http://github.com/minishift/minishift-centos/minishift-centos.iso", true},
+		{"https://github.com/minishift//minishift/minishift-b2d.iso", true},
+		{"file:///home/foo/bar/bar.iso", true},
```

Comment:
How is this supposed to pass? You usually don't have this path on your file system. You will need to create a tmp directory and touch an iso file in there. Then write your test against it. See the other test cases which do similar things.